package-resources.yml

```diff
@@ -0,0 +1,54 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: Package
+metadata:
+  creationTimestamp: null
+  name: kapp-controller.carvel.dev.0.0.0
+spec:
+  refName: kapp-controller.carvel.dev
+  releasedAt: null
+  template:
+    spec:
+      deploy:
+      - kapp: {}
+      fetch:
+      - git: {}
+      template:
+      - ytt:
+          paths:
+          - config
+          - config-release
+      - kbld: {}
+  valuesSchema:
+    openAPIv3: null
+  version: 0.0.0
+
+---
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: PackageMetadata
+metadata:
+  creationTimestamp: null
+  name: kapp-controller.carvel.dev
+spec:
+  displayName: kapp-controller
+  longDescription: kapp-controller.carvel.dev
+  shortDescription: kapp-controller.carvel.dev
+
+---
+apiVersion: packaging.carvel.dev/v1alpha1
+kind: PackageInstall
+metadata:
+  annotations:
+    kctrl.carvel.dev/local-fetch-0: .
+  creationTimestamp: null
+  name: kapp-controller
+spec:
+  packageRef:
+    refName: kapp-controller.carvel.dev
+    versionSelection:
+      constraints: 0.0.0
+  serviceAccountName: kapp-controller-sa
+status:
+  conditions: null
+  friendlyDescription: ""
+  observedGeneration: 0
+  
```

Comment:
nit: Is there space/tab present on the last line? (because there's the `No newline at end of file` present after it)